node_modules/oae-core/documentpreview/js/documentpreview.js

```diff
@@ -164,6 +164,17 @@ define(['jquery', 'underscore', 'oae.core', 'lazyload'], function($, _, oae) {
                     return callback({'code': jqXHR.status, 'msg': jqXHR.responseText});
                 }
             });
+
+            // Request the page content text version
+            $.ajax({
+                'url': constructDocumentPreviewURL('page.' + page.pageNumber + '.txt'),
+                'crossDomain': true,
```

Comment:
@nicolaasmatthijs:  Here's the [commit](https://github.com/oaeproject/3akai-ux/commit/db763f97770fd3279a9c51d9a469a14f99c037f9#diff-c9a9baad3b5d7cace1b766de4d0e2f47) ("Removed text preview and added crossDomain flag for Markdown preview"). I'm still not sure why this is needed, but maybe this will jog your memory?